.changeset/neat-lamps-press.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/core-components': patch
+'@backstage/plugin-splunk-on-call': patch
+'@backstage/plugin-xcmetrics': patch
+'@backstage/plugin-techdocs': patch
+'@backstage/plugin-catalog': patch
+---
+
+MUI v5 fix: express theme spacing without extra unit
```

Comment:
They prefer we use "Material UI" (using MUI is when we refer to the Org)

Reply:
Alright!